XRTK-Core/Packages/com.xrtk.core/StandardAssets/Shaders/MixedRealityWireframe.shader.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 4a2ab4faa2fc4a009bcfdf058d1e0659
+guid: 5c1653eb4e20b76499141de7bc57c063
```

Comment:
Not sure I like the guid change either. This could break existing wireframe shader references.

Reply:
It's meant to break the old shader references as it's effectively a new shader